docs/sources/reference/config-blocks/remotecfg.md

```diff
@@ -43,6 +43,23 @@ Name             | Type                 | Description
 `attributes`     | `map(string)`        | A set of self-reported attributes.                | `{}`        | no
 `poll_frequency` | `duration`           | How often to poll the API for new configuration.  | `"1m"`      | no
 `name`           | `string`             | A human-readable name for the collector.          | `""`        | no
+`bearer_token_file`      | `string`            | File containing a bearer token to authenticate with.          |         | no
+`bearer_token`           | `secret`            | Bearer token to authenticate with.                            |         | no
+`enable_http2`           | `bool`              | Whether HTTP2 is supported for requests.                      | `true`  | no
+`follow_redirects`       | `bool`              | Whether redirects returned by the server should be followed.  | `true`  | no
+`proxy_url`              | `string`            | HTTP proxy to send requests through.                          |         | no
+`no_proxy`               | `string`            | Comma-separated list of IP addresses, CIDR notations, and domain names to exclude from proxying. | | no
+`proxy_from_environment` | `bool`              | Use the proxy URL indicated by environment variables.         | `false` | no
+`proxy_connect_header`   | `map(list(secret))` | Specifies headers to send to proxies during CONNECT requests. |         | no
+
+ At most, one of the following can be provided:
+ - [`bearer_token` argument](#endpoint-block).
+ - [`bearer_token_file` argument](#endpoint-block).
```

Comment:
Are these the right links? I don't see an `endpoint` block in this doc

Reply:
Whoops, fixed. I now have them point to the arguments table.